### PR TITLE
vox oxygen toxicity, respiration changes

### PR DESCRIPTION
**Pull request by Errant-4**
## About the PR
We all knew this was coming, but it's coming today! (Maybe)

Vox take poison damage from oxygen and die in normal station air in 2 minutes. They get a TOXIC alert when breathing oxygen (this could be switched for the more specific "too much oxygen" alert icon, but I don't really see the need to give each toxic gas their own distinct alert)
They must breathe pure nitrogen (or any other nitrogen-containing but oxygen-free gas mixture, if those somehow exist somewhere)
Since it requires taking off your mask, they can't eat and drink without taking damage unless they have access to a nitrogen-only environment, but they can survive a meal in oxygen. 
Eating your emergency cube and drinking your emergency drink, then immediately turning internals back on results in about 21 poison damage. 2 dylo pills basically clear that up, although that means taking off the mask again briefly so might want to add a charcoal pill for good measure. Or preferably just ask for the dylovene to be injected instead.

Note that a vox succumbing to oxygen toxicity will spend a human-equivalent time in the crit state and will after dying still be revivable, because in crit state mobs stop breathing. Thus about half of their damage will be airloss instead of toxin.

Vox now exhale ammonia. Due to this, breathing ammonia is now more dangerous to them (since it asphyxiates them even in moderate quantities), while CO2 is now harmless. 

New spawnable vox variant that is already equipped with breathing gear and N2 tank, for admin ease of use (note that currently the internals must still be activated by hand, if they are spawned in an oxygen atmosphere and they will likely take a little poison damage even if you do so immediately)

## Why / Balance
There has been somewhat of an increase in admeme voxspawns, so lets not establish their current respiratory details as the normal. Vox being hurt by oxygen is in line with how vox are expected to be, based on SS13 (where they apparently die EVEN FASTER)

Ammonia was honestly just a result of me being bothered by the fact that if they would exhale either CO2 or N2O, where would the oxygen come from in high and constant enough quantities to be constantly exhaled? Plus trashbirds being naturally stimky (as far as humans are concerned) is also kinda funny. In practice, it will likely never actually matter due to vox either breathing N2 safely from a bottle, or dying to station oxygen. As far as others are concerned, the small amounts of ammonia will be handled by the scrubber system same as CO2 would be. This is mostly just a thematic change

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Errant
- tweak: Vox now take significant damage over time while inhaling oxygen.

ADMIN:
- add: When spawning vox for admemes, they should be given/spawned in a nitrogen-filled space if possible. If they must be placed in normal air, an alternate vox prototype now comes with breathing gear and N2 tank already equipped, but the internals must be turned on via right click and even if you do this instantly they will still probably take ~4 poison damage from the air, so heal them afterwards.

### CODE REVIEW

**Comment by LankLTE** [comment]
Probably doesn’t need a normal CL since Vox currently aren’t player facing at all

**Comment by Errant-4** [comment]
ith vox. 

> Probably doesn’t need a normal CL since Vox currently aren’t player facing at all

Well since they don't have AI, the only way they would ever be in play is under the control of players. They are just *very rare* due to being admeme only. But I felt players ought to know about a change this lethal both for when picking the ghostrole and when encountering them. I will remove the player CL if there is consensus its not necessary/admin will have to brief them anyway.

**Comment by brainfood1183** [comment]
Shouldnt Vox spawn in with a face mask and nitrogen pack already setup?

**Comment by Errant-4** [comment]
> Shouldnt Vox spawn in with a face mask and nitrogen pack already setup?

I only mentioned it in the admin CL, I did add a vox prototype that has breathing gear equipped. But it needs to be toggled on by hand after spawning.
I need #25083 for auto-activation

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml [hunk 1]
```diff
@@ -342,3 +342,11 @@
...
     jumpsuit: ClothingUniformJumpsuitClownBanana
     mask: ClothingMaskClownBanana
+
+#Vox Survival Gear
+- type: startingGear
+  id: VoxBreathingGear
+  equipment:
+    outerClothing: ClothingOuterVest
+    mask: ClothingMaskBreath
+    suitstorage: NitrogenTankFilled
```
Comment (metalgearsloth) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:352]: Just make it use species loadouts.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**